test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -225,8 +225,11 @@ func accumulateNodeAvailableResources(nrt nrtv1alpha2.NodeResourceTopology, reas
 	klog.Infof("resInfoList available %s: %s", reason, e2enrt.ResourceInfoListToString(resInfoList))
 	return resInfoList, nil
 }
-func SaturateZoneUntilLeft(zone nrtv1alpha2.Zone, requiredRes corev1.ResourceList) (corev1.ResourceList, error) {
+func SaturateZoneUntilLeft(zone nrtv1alpha2.Zone, requiredRes corev1.ResourceList, filter func(*corev1.ResourceList)) (corev1.ResourceList, error) {
 	paddingRes := make(corev1.ResourceList)
+
+	filter(&requiredRes)
```

Comment:
because this is a map, we should return the mutated `ResourceList` and use the mutated one

Reply:
thanks for pointing out this, I have updated this block in https://github.com/openshift-kni/numaresources-operator/pull/942/commits/c4b438f3b2a9755c77ef7971f6a3f92ce95768ef because the proposed changes are needed by new tests